src/lib/features/segment/segment-store.test.ts

```diff
@@ -48,7 +48,7 @@ describe('usage counting', () => {
         await db.rawDatabase.table('change_requests').delete();
     });
 
-    test('segment usage in active CRs is counted iff we ask for it', async () => {
+    test('segment usage in active CRs is counted if we ask for it', async () => {
```

Comment:
This wasn't actually a typo, but shorthand for "if and only if", but maybe I should've written it out? 😅